README.md

```diff
@@ -4,6 +4,7 @@
 [![Coverage Status](https://coveralls.io/repos/github/nearform/graphql-hooks/badge.svg?branch=master)](https://coveralls.io/github/nearform/graphql-hooks?branch=master)
 ![](https://img.shields.io/bundlephobia/minzip/graphql-hooks.svg?style=flat)
 [![npm version](https://badge.fury.io/js/graphql-hooks.svg)](https://badge.fury.io/js/graphql-hooks)
+[![All Contributors](https://img.shields.io/badge/all_contributors-4-orange.svg?style=flat-square)](#contributors)
```

Comment:
This is hardcoded to 4, is it updated by the bot?

Reply:
It's updated by the bot / npm commands automatically 🙌